Automattic/woocommerce-payments
# Card readers branding settings UI spacing adjustments

#### Changes proposed in this Pull Request

Spacing adjustment for color picker and file upload preview components.

Before:
<img width="165" alt="Screenshot 2022-03-18 at 14 03 56" src="https://user-images.githubusercontent.com/8667118/158999922-06cca050-bad5-49c1-8598-e5a98cb716dd.png">
<img width="554" alt="Screenshot 2022-03-18 at 14 04 07" src="https://user-images.githubusercontent.com/8667118/158999933-f27bb2ea-c9bb-4386-a494-0ce925c77094.png">

After:
<img width="167" alt="Screenshot 2022-03-18 at 14 00 19" src="https://user-images.githubusercontent.com/8667118/158999658-7b4a9e25-be4d-4786-916f-2c354ab1865b.png">
<img width="556" alt="Screenshot 2022-03-18 at 14 00 34" src="https://user-images.githubusercontent.com/8667118/158999684-9ce85f9b-2564-418d-9d0d-bb0c548db92b.png">

#### Testing instructions
- Update the `_wcpay_feature_card_readers` option to `1`.
- Update the `card_present_eligible` to `true`.
- Update `isBrandingEnabled` feature flag to `true` in `client/card-readers/settings/index.js`.
- Navigate to `Payments` -> `Card Readers page` -> `Receipt details`.
- Verify file upload preview and color picker components have correct spacings.
-------------------
